src/main/java/nextstep/fp/StreamStudy.java

```diff
@@ -27,7 +28,14 @@ public static void printLongestWordTop100() throws IOException {
                 .get("src/main/resources/fp/war-and-peace.txt")), StandardCharsets.UTF_8);
         List<String> words = Arrays.asList(contents.split("[\\P{L}]+"));
 
-        // TODO 이 부분에 구현한다.
+        words.stream()
+                .filter(n -> n.length() > 12)
+                .sorted(Comparator.comparing(String::length).reversed())
+                .map(String::toLowerCase)
+                .distinct()
+                .limit(100)
+                .forEach(System.out::println);
```

Comment:
```suggestion
        words.stream()
                .filter(n -> n.length() > 12)
                .map(String::toLowerCase)
                .distinct()
                .sorted(Comparator.comparing(String::length).reversed())
                .limit(100)
                .forEach(System.out::println);
```

`sorted` 하기전에 좀 더 정제하는것도 좋을것 같아요! 🤔